src/context-items/standard-validation.ts

```diff
@@ -10,12 +10,18 @@ export class StandardValidation implements ContextItem {
     private readonly validator: StandardValidator,
     private readonly negated: boolean,
     private readonly options: any[] = [],
+    // For testing only.
+    // Deliberately not calling it `toString` in order to not override `Object.prototype.toString`.
+    private readonly stringify = toStringImpl,
   ) {}
 
   async run(context: Context, value: any, meta: Meta) {
-    const result = this.validator(toString(value), ...this.options);
-    if (this.negated ? result : !result) {
-      context.addError(this.message, value, meta);
-    }
+    const values = Array.isArray(value) ? value : [value];
```

Comment:
@fedeci This row is causing issue for some validation.
For example, isInt, IsIn validators are not validating correctly for empty array value.
```
const validator = checkSchema({ field: { isInt:{} }});
const req = { body: { field: [] } };
await validator.run(req);
const errors = validationResult(req);

```
For above code, errors shouldn't be empty. But errors become empty because of above row.
Any idea?

Reply:
Yes, it was meant to be like that. If you want to validate each item of the array you need to use
```js
const validator = checkSchema({ 'field.*': { isInt:{} }});
```

Reply:
No, `field` should be number. But there can be case that `field` in request is array. In that case, error should be triggered.

Reply:
I think it is bug. Right? @fedeci
Any idea to fix?